assets/NonFunctionalRequirements

```diff
@@ -0,0 +1,8 @@
+Kwaliteitseisen
+
+Non-functionele eisen:
+1. Gebruikersgegevens moeten beveiligd zijn
+2. Het systeem moet bruikbaar zijn voor alle medewekers/admins tegelijk
```

Comment:
Hier is het handig om een nummer te specificeren, zodat het makkelijk testbaar wordt.